LR_Tourist/TouristWebApp/Areas/Identity/IdentityHostingStartup.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.UI;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using TouristWebApp.Data;
+
+[assembly: HostingStartup(typeof(TouristWebApp.Areas.Identity.IdentityHostingStartup))]
+namespace TouristWebApp.Areas.Identity
+{
+    public class IdentityHostingStartup : IHostingStartup
+    {
+        public void Configure(IWebHostBuilder builder)
+        {
+            builder.ConfigureServices((context, services) => {
+                services.AddDbContext<TouristWebAppContext>(options =>
+                    options.UseSqlServer(
+                        context.Configuration.GetConnectionString("TouristWebAppContextConnection")));
+               // .AddIdentity<IdentityUser, IdentityRole>()
```

Comment:
It should be in the common Startup file.

Reply:
fixed